project/sites/parolecomni/themes/parolecomni_theme/src/scss/components.scss

```diff
@@ -5,6 +5,29 @@
 @import 'init/index';
 @import '../../node_modules/nicsdru_unity_theme/src/scss/3_components/index';
 
+#main-menu li {
+  border-right: 1px solid #fff;
+}
+
+#main-menu .menu .menu-item .active, #main-menu .menu .menu-item .active + .menu-main_sub .title a
+{
+  background: none;
+  border-bottom: none;
+}
+
+.front {
+
+  h1 {
+    font-size: 2.2rem;
```

Comment:
I don't think we should be changing heading sizes or colours from what is in the Unity theme.

Reply:
This is now updated with font-sizes on h1 and h2 removed.